drivers/pwm/pwm_ite_it8xxx2.c

```diff
@@ -13,16 +13,26 @@
 #include <errno.h>
 #include <kernel.h>
 #include <soc.h>
+#include <soc_dt.h>
 #include <stdlib.h>
 
 #include <logging/log.h>
 LOG_MODULE_REGISTER(pwm_ite_it8xxx2, CONFIG_PWM_LOG_LEVEL);
 
 #define PWM_CTRX_MIN	100
-#define PWM_EC_FREQ	MHZ(8)
+#define PWM_FREQ	EC_FREQ
```

Comment:
I get compile errors when compiling this code:

```
ERROR: [/mnt/host/source/src/platform/ec/build/zephyr/hayato:ro]/home/jrosenth/zephyr/drivers/pwm/pwm_ite_it8xxx2.c:23:25: error: 'EC_FREQ' undeclared (first use in this function); did you mean 'ECREG'?
ERROR: [/mnt/host/source/src/platform/ec/build/zephyr/hayato:ro]   23 | #define PWM_FREQ        EC_FREQ
ERROR: [/mnt/host/source/src/platform/ec/build/zephyr/hayato:ro]      |                         ^~~~~~~
ERROR: [/mnt/host/source/src/platform/ec/build/zephyr/hayato:ro]/home/jrosenth/zephyr/drivers/pwm/pwm_ite_it8xxx2.c:80:38: note: in expansion of macro 'PWM_FREQ'
ERROR: [/mnt/host/source/src/platform/ec/build/zephyr/hayato:ro]   80 |                 *cycles = (uint64_t) PWM_FREQ;
ERROR: [/mnt/host/source/src/platform/ec/build/zephyr/hayato:ro]      |                                      ^~~~~~~~
ERROR: [/mnt/host/source/src/platform/ec/build/zephyr/hayato:ro]/home/jrosenth/zephyr/drivers/pwm/pwm_ite_it8xxx2.c:23:25: note: each undeclared identifier is reported only once for each function it appears in
ERROR: [/mnt/host/source/src/platform/ec/build/zephyr/hayato:ro]   23 | #define PWM_FREQ        EC_FREQ
ERROR: [/mnt/host/source/src/platform/ec/build/zephyr/hayato:ro]      |                         ^~~~~~~
ERROR: [/mnt/host/source/src/platform/ec/build/zephyr/hayato:ro]/home/jrosenth/zephyr/drivers/pwm/pwm_ite_it8xxx2.c:80:38: note: in expansion of macro 'PWM_FREQ'
ERROR: [/mnt/host/source/src/platform/ec/build/zephyr/hayato:ro]   80 |                 *cycles = (uint64_t) PWM_FREQ;
ERROR: [/mnt/host/source/src/platform/ec/build/zephyr/hayato:ro]      |                                      ^~~~~~~~
```

I don't see any references to `EC_FREQ` in the tree. What am I missing?

Reply:
The EC_FREQ is defined in tachometer PR (https://github.com/zephyrproject-rtos/zephyr/pull/39993), I think that tachometer PR is ready to merge.

Reply:
This suggests that this driver is not getting built in CI. @RuibinChang will you please look into enabling `tests/drivers/pwm/` for an ITE board?

Reply:
I build board it8xxx2_evb with pwm driver, it doesn't show the error.